test/unit/devices_repository_test.dart

```diff
@@ -158,17 +170,24 @@ void main() {
               }
             }
           },
+          {
+            "__typename": "NewUnknownTrait",
```

Comment:
Any background on this?

Reply:
It's to make sure our code can parse any currently-unsupported Trait as the UnknownTrait class